pkg/controller/util.go

```diff
@@ -240,11 +240,22 @@ func newScratchPersistentVolumeClaimSpec(pvc *v1.PersistentVolumeClaim, pod *v1.
 		LabelImportPvc:   pvc.Name,
 	}
 
+	annotations := make(map[string]string, 0)
+	// Copy kubevirt.io annotations, but NOT the CDI annotations as those will trigger another import/upload/clone on the scratchspace
+	// pvc.
+	if len(pvc.GetAnnotations()) > 0 {
+		for k, v := range pvc.GetAnnotations() {
+			if strings.Contains(k, common.KubeVirtAnnKey) && !strings.Contains(k, common.CDIAnnKey) {
```

Comment:
This is probably fine, but I just wonder:  if we only really care about `kubevirt.io/provisionOn` why not just limit to that?